### PR TITLE
adding Code

### PR DESCRIPTION
**Pull request by mood-duck**

### CODE REVIEW

**Comment by roman-gherman** [comment]
1. Branch name trebuie sa fie mai sugestiv, gen: BinaryGap
2. Commit message/comment - trebuie sa scrii in comment ce ai facut, daca e primul commit, poate sa fie "Binary Gap - did the solution to 93%"

File: BinaryGap/Program.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    public int solution(int N)
+    {
+        string numberConvertedToBinary = Convert.ToString(N, 2);
+        int max = 0;
+        bool isCounting = false;
+        int count = 0;
+        
+        for (int i = 0; i < numberConvertedToBinary.Length; i++)
+        {
+
```
Comment (roman-gherman) [inline BinaryGap/Program.cs:18]: remove this space

File: BinaryGap/Program.cs [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        bool isCounting = false;
+        int count = 0;
+        
+        for (int i = 0; i < numberConvertedToBinary.Length; i++)
+        {
+            
+            if (numberConvertedToBinary[i] == '1')
+            {
+                isCounting = true;
+                if (count >= max )
```
Comment (roman-gherman) [inline BinaryGap/Program.cs:22]: add a new line before If statement and variable assignment

File: BinaryGap/Program.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            }
+            else
+            {
+                count += 1;
+            }
+        }
+        
+        return max;
+    }
+    static public void Main(String[] args)
```
Comment (roman-gherman) [inline BinaryGap/Program.cs:36]: add a new line before Main

File: BinaryGap/Program.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    {
+        Solution a = new Solution();
+        a.solution(6 );
```
Comment (roman-gherman) [inline BinaryGap/Program.cs:39]: a.solution(6 ); remove the space

**Comment by denisburlacu** [comment]
- add sln
- move class in separate file

File: Codility/BinaryGap/BinaryGap.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+// you can also use other imports, for example:
+// using System.Collections.Generic;
+
+// you can write to stdout for debugging purposes, e.g.
+// Console.WriteLine("this is a debug message");
+
+class BinaryGap {
+    public int solution(int N)
```
Comment (denisburlacu) [inline Codility/BinaryGap/BinaryGap.cs:9]: rename this to CamelCase please

File: Codility/BinaryGap/BinaryGap.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        string numberConvertedToBinary = Convert.ToString(N, 2);
+        int max = 0;
+        bool isCounting = false;
+        int count = 0;
+        
+        for (int i = 0; i < numberConvertedToBinary.Length; i++)
+        {
+
```
Comment (denisburlacu) [inline Codility/BinaryGap/BinaryGap.cs:18]: no spaces between for and if please

**Pull request merged**